pkg/portal/regions.go

```diff
@@ -72,7 +74,8 @@ func (p *portal) regions(w http.ResponseWriter, r *http.Request) {
 	}
 
 	if value, found := os.LookupEnv("AZURE_ENVIRONMENT"); found {
-		if value == azureclient.PublicCloud.Environment.Name {
+		// AZURE_ENVIRONMENT variable can either be AZUREPUBLICCLOUD or AZUREUSGOVERNMENTCLOUD
+		if value == strings.ToUpper(azureclient.PublicCloud.Environment.Name) {
```

Comment:
We should use `strings.EqualFold()` here, instead of relying on upping the case.

Reply:
Fixed!